src/views/PeopleSearch/index.js

```diff
@@ -415,59 +382,19 @@ class PeopleSearch extends Component {
   }
 
   async updateURL() {
-    let searchParameters = '?';
-    searchParameters += this.state.firstNameSearchValue
-      ? `firstName=${this.state.firstNameSearchValue}&`
-      : '';
-    searchParameters += this.state.lastNameSearchValue
-      ? `lastName=${this.state.lastNameSearchValue}&`
-      : '';
-    searchParameters += this.state.majorSearchValue ? `major=${this.state.majorSearchValue}&` : '';
-    searchParameters += this.state.minorSearchValue ? `minor=${this.state.minorSearchValue}&` : '';
-    searchParameters += this.state.hallSearchValue ? `hall=${this.state.hallSearchValue}&` : '';
-    searchParameters += this.state.classTypeSearchValue
-      ? `classType=${this.state.classTypeSearchValue}&`
-      : '';
-    searchParameters += this.state.homeCitySearchValue
-      ? `homeCity=${this.state.homeCitySearchValue}&`
-      : '';
-    searchParameters += this.state.stateSearchValue ? `state=${this.state.stateSearchValue}&` : '';
-    searchParameters += this.state.countrySearchValue
-      ? `country=${this.state.countrySearchValue}&`
-      : '';
-    searchParameters += this.state.departmentSearchValue
-      ? `department=${this.state.departmentSearchValue}&`
-      : '';
-    searchParameters += this.state.buildingSearchValue
-      ? `building=${this.state.buildingSearchValue}&`
-      : '';
-    searchParameters += this.state.includeAlumniSearchValue
-      ? `includeAlumni=${this.state.includeAlumniSearchValue}&`
-      : '';
+    const searchParameters = Object.entries(this.state.searchValues)
+      .map(([key, value]) => (value ? `${key}=${value}` : '')) // [ 'firstName=value', 'state=texas']
+      .filter((n) => n) // removes empty strings
```

Comment:
It might be slightly more performant to filter and then map, although with an array of size 12 it's unlikely to matter

Reply:
Also, the comment is somewhat redundant in my opinion.